lib/Importer/ONNXModelLoader.cpp

```diff
@@ -544,6 +544,20 @@ bool ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
     return true;
   }
 
+  if (typeName == "LengthsSum") {
```

Comment:
Could you move this code and Caffe2 loader's code to `CommonOperatorLoader.h`?

Reply:
I put it there initially, but then I moved it out to the two separate loaders because I added a check that the dimensions of the `lengths` input is 1 in the ONNX loader at Roman's request. The Caffe2 protobuf doesn't specify input sizes as far as I know so this is not applicable there.

Reply:
Caffe2 and ONNX protos are really very similar. In fact, ONNX format is re-branded Caffe2 proto :)

They both specify the same amount of information. We have static shapes in Glow, so `lengths` in your code is output of some other node (probably Variable). You always know its number of dims. So I don't see why this code can't be shared.

Now as for moving assertion here. I guess this question is bigger than this PR. `len(lengths) > 1` totally does not make sense, it means the input is broken. Should we do this validation or should this be done on higher level? And we'd just assert? I'm sure there are move such places, other than this one. @rdzhabarov

Reply:
> They both specify the same amount of information. We have static shapes in Glow, so lengths in your code is output of some other node (probably Variable). You always know its number of dims. So I don't see why this code can't be shared.

Code sharing is fine as long as we can report from the shared code that parsing of certain operator failed.

Reply:
I think there is, but I'll just have to break convention and write a parser for this op that returns `bool` instead of `void` in `CommonOperatorLoader.h`.

Reply:
It's fine to break this convention. We will have to break it later anyway.